src-ghc/Pact/Server/ApiServer.hs

```diff
@@ -56,96 +59,101 @@ data ApiEnv = ApiEnv
   }
 makeLenses ''ApiEnv
 
-type Api a = ReaderT ApiEnv Snap a
+type ApiT a = ReaderT ApiEnv (ExceptT ServantErr IO) a
```

Comment:
I don't really like the `T` as it's not a transformer, not worth slowing up the job though. Good work!

Reply:
Makes sense, I've changed it back to `Api`.